test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Ensure none of the build config inject secrets still exist in the file system
+for s in $(ls /tmp/secret? secret?); do
+    if [ -s "${s}" ]; then
+        echo "Found secret file which should have been removed: ${s}"
+        exit 1
```

Comment:
You're right, of course. I had actually determined that and forgotten it. To answer your question, the 'exit 1' line is what triggers the test case explosion. The echo that precedes it is informational. The exit aborts the script and prevents the desired output from being captured. Is that acceptable?